pages/submission/PKPSubmissionHandler.inc.php

```diff
@@ -222,6 +224,6 @@ function setupTemplate($request) {
 	 * @return int
 	 */
 	abstract function getStepCount();
-}
+		}
```

Comment:
Watch out for accidental whitespace changes.

Reply:
Yep, dunno where this is coming from, i will fix it.

Reply:
Fixed.